js/src/pubsub.js

```diff
@@ -13,6 +13,11 @@ const whilst = require('async/whilst')
 const each = require('async/each')
 const hat = require('hat')
 
+let isWindows = false
+if (process.platform && process.platform === 'win32') {
```

Comment:
```
const isWindows = process.platform && process.platform === 'win32'
```

Doesn't standard warn you here that this is a uneccessary conditional?

Reply:
Good catch. I didn't get a warning but fixed it nonetheless.